fuseki/sample-ubd.trig

```diff
@@ -46,21 +46,21 @@ graph <cube-project/ubd/dataset> {
 
 <cube-project/ubd/dimensions-metadata> void:inDataset <ubd-example> .
 graph <cube-project/ubd/dimensions-metadata> {
-  <cube-project/ubd/dimensions-metadata> a cc:DimensionsMetadataCollection ;
-    schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/dimension%20year> ;
-    schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/station> ;
-    schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/aggregation> ;
-    schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/pollutant> ;
-    schema:hasPart <cube-project/ubd/dimensions-metadata?dimension=https://environment.ld.admin.ch/foen/ubd/28/unit-id> ;
+  <cube-project/ubd/dimensions-metadata> a cc:DimensionMetadataCollection , hydra:Resource ;
+    schema:hasPart <cube-project/ubd/dimensions-metadata/dimension-year> ;
+    schema:hasPart <cube-project/ubd/dimensions-metadata/station> ;
+    schema:hasPart <cube-project/ubd/dimensions-metadata/aggregation> ;
+    schema:hasPart <cube-project/ubd/dimensions-metadata/pollutant> ;
+    schema:hasPart <cube-project/ubd/dimensions-metadata/unit-id> ;
```

Comment:
So you are using the slugified column name for the dimension? It will add a random value at the end right?

Reply:
Technically it can be anything. Do we have to make it forcefully unique? I thought to simply append the `targetProperty` string or shrunk URI like `cube-project/ubd/dimensions-metadata/schema:age`

Reply:
Yeah that works for me. I just wanted to make sure you were not intending to use the CSV column name because one column can definitely be mapped to multiple properties.